sqlmesh/core/config/connection.py

```diff
@@ -66,7 +66,11 @@ def validate(cls: t.Any, data: t.Any) -> t.Any:
             return data
         try:
             importlib.import_module(import_name)
-        except ImportError:
+        except ImportError as e:
+            if debug_mode_enabled():
+                raise
+
+            logger.error(str(e))
```

Comment:
This swallows the original stacktrace. Maybe
```
logger.exception("Failed to import the engine library")
```
`exception` will pick up the correct exception from the context, no need to explicitly pass `e`.

Reply:
Perfect, TIL!